packages/xchain-binance/__tests__/client.test.ts

```diff
@@ -164,8 +273,11 @@ describe('BinanceClient Test', () => {
     const beforeFreeze = await client.getBalance()
     expect(beforeFreeze.length).toEqual(1)
 
+    await delay(1000) //delay before transaction
```

Comment:
Same question here (and for all other delays we have in this test file): Is this delay still needed?